camunda-formio-tasklist-vue/.eslintrc.js

```diff
@@ -15,7 +15,6 @@ module.exports = {
     "no-console": "warn",
     "no-debugger": process.env.NODE_ENV === "production" ? "warn" : "off",
     "vue/no-unused-components": "warn",
-    "space-before-function-paren": 1,

```

Comment:
any reason to remove this?

Reply:
https://github.com/AOT-Technologies/forms-flow-ai-extensions/blob/generic-search-component/camunda-formio-tasklist-vue/.eslintrc.js#L28

The rule was already existing, since the rule was duplicated on `npm run lint` an error was being thrown. That's why decided to remove the duplicate code

Reply:
Note to me: 1 was warning, 2 for error and 0 for off